api_tests/comments/views/__init__.py

```diff
@@ -0,0 +1 @@
+__author__ = 'Saman'
```

Comment:
We prefer not to use `__author__` to provide attribution of code; we have other means of providing credit. Can you remove this?